src/ApiPort.VisualStudio.Common/Analyze/ProjectAnalyzer.cs

```diff
@@ -117,5 +125,25 @@ private IEnumerable<string> GetAllAssembliesInDirectory(string directory)
                 return Enumerable.Empty<string>();
             }
         }
+
+        private IEnumerable<string> GetPackageReferences(Project project)
+        {
+            var componentModel = (IComponentModel)Microsoft.VisualStudio.Shell.ServiceProvider.GlobalProvider.GetService(typeof(SComponentModel));
+            IVsPackageInstallerServices installerServices = componentModel.GetService<IVsPackageInstallerServices>();
```

Comment:
Nitpicks: I suggest `using Microsoft.VisualStudio.Shell` and `var installerServices`

Reply:
This would lead to ambiguities: error CS0104: 'Task' is an ambiguous reference between 'Microsoft.VisualStudio.Shell.Task' and 'System.Threading.Tasks.Task'

Reply:
I would use dependency injection through the constructor to get IVsPackageInstallerServices instead of accessing global services. This will make it easier to test the code because there are no hidden dependencies.